controllers/reconciler.go

```diff
@@ -333,6 +348,41 @@ func (r *SingleClusterReconciler) updateStatus() error {
 	return nil
 }
 
+func (r *SingleClusterReconciler) updateAccessControlStatus() error {
+	if r.aeroCluster.Spec.AerospikeAccessControl == nil {
+		return nil
+	}
+
+	r.Log.Info("Update access control status for AerospikeCluster")
+
+	// Get the old object, it may have been updated in between.
+	newAeroCluster := &asdbv1beta1.AerospikeCluster{}
+	if err := r.Client.Get(
+		context.TODO(), types.NamespacedName{
+			Name: r.aeroCluster.Name, Namespace: r.aeroCluster.Namespace,
+		}, newAeroCluster,
+	); err != nil {
+		return err
+	}
+
+	// AerospikeAccessControl
+	statusAerospikeAccessControl := &asdbv1beta1.AerospikeAccessControlSpec{}
+	lib.DeepCopy(
+		statusAerospikeAccessControl, r.aeroCluster.Spec.AerospikeAccessControl,
+	)
+
+	newAeroCluster.Status.AerospikeClusterStatusSpec.AerospikeAccessControl = statusAerospikeAccessControl
```

Comment:
Can we add a check here to update the `AerospikeAccessControl` only when there is a diff between spec and status. This avoid patchStatus calls